### ISSUE TITLE
Check product data

### ISSUE THREAD

**Issue by pmayd** [CorrelAidSwitzerland/a4d#86]

**Comment by pmayd** [comment]
@tvolegtv I have filled in some details. But I need the schema of the data before I can ingest it in the database, see slack

**Comment by pmayd** [comment]
Ingest was successful, you should see the table now!

**Comment by tvolegtv** [comment]
where is it? google cloud?

**Comment by pmayd** [comment]
No, in big query as that is the one we want to check so you have to open the big query service in the console and look at the data in the table product_data

**Comment by pmayd** [comment]
@tvolegtv was is the status here and can you try to complete/close this issue by the end of the year?

**Comment by tvolegtv** [comment]
@pmayd  I think we can close this.
These checks seem to be regular as soon as we have new data - perhaps there is no point in keeping it as an open task.
